pkg/cmd/pulumi/convert.go

```diff
@@ -46,11 +49,11 @@ func newConvertCmd() *cobra.Command {
 
 			var programGenerator programGeneratorFunc
 			switch language {
-			case langDotnet:
+			case "c#":
```

Comment:
This should definitely allow "csharp" as well. We use that else where for template and example names.